apps/hash-graph/bin/cli/src/subcommand/server.rs

```diff
@@ -314,3 +328,20 @@ pub async fn server(args: ServerArgs) -> Result<(), GraphError> {
 
     Ok(())
 }
+
+pub async fn healthcheck(address: ApiAddress) -> Result<(), GraphError> {
+    let request_url = format!(
+        "http://{}:{}/api-doc/openapi.json",
+        address.api_host, address.api_port
+    );
```

Comment:
I don't know if there is a better option, I'm not particularly happy about `http`. We could use the `router` instead, but this also requires connecting the pool.

Reply:
I think it's fine as we wouldn't be setting up TLS from the container but have a reverse proxy in front with TLS termination anyway. Although we may want to introduce a `mTLS` ceritficate or some sort of Origin cert at some point. This is fine for now :+1: